util/ci/arch-testscript

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash -le
+
+#-l above (login shell) so that /etc/profile runs so that perl is
+# on the path.
+
+#-e so that if any command has an exit code != 0, this script will
+#  exit immediately.
+
+# Python is python 3, but gnucash doesn't work with python 3. There
+# doesn't seem to be a way to tell automake to use /usr/bin/python2,
+# so we'll mock this up
+mkdir autotools_bin;
+ln -s /usr/bin/python2 autotools_bin/python;
+export PATH=/autotools_bin:"$PATH";
+echo path is "$PATH";
+echo python version is "$(python --version)";
+git clone --depth 1 https://github.com/Gnucash/gnucash.git;
```

Comment:
With this clone, you won't be using the repository under test (you'll get whatever is currently on master).

Reply:
Yikes good catch.

Reply:
For what it's worth, I don't think it would always be checking `master`, but it would always be checking whatever is `HEAD` in github. Not quite as bad, but still quite bad. I have a fix committed now, although I'll close this pull request and open a new one to clean things up.